src/components/gear/cyberware/CyberlimbAttributeComponent.js

```diff
@@ -1,26 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ModificationButton from '../../ModificationButton';
 
-const CyberlimbAttribute = ({incrementAttribute, decrementAttribute, attribute, att}) => {
+const CyberlimbAttribute = ({incrementAttribute, decrementAttribute, attributeValue, attributeName}) => {
```

Comment:
Way more logical naming convention. Don't know why I didn't think of it myself.

Reply:
Naming things well the first time is probably one of the hardest things to do.  Also switching between stuff, I kept losing track of what they meant, so I did it for my forgetful brain.